src/run-tests.ts

```diff
@@ -20,6 +21,8 @@ export interface IRunTestsOptions {
   reporter?: string | undefined;
   ui?: string | undefined;
   timeout?: number | undefined;
```

Comment:
@AviVahl what's with the | undefined?

Reply:
https://www.typescriptlang.org/tsconfig#exactOptionalPropertyTypes